stories/components/Field.js

```diff
@@ -62,5 +62,42 @@ storiesOf('Components', module)
       >
         <Field isReadOnly fieldId="987" />
       </StoryItem>
+
+      <StoryItem
+        title="Field (Editing)"
+        description="A field component when in an editing state"
+      >
+        <Field
+          fieldId="123"
+          label="Field label text"
+          instructions="Instruction text to help inform the user of what they are mean't to add into this field."
+          isStructureEditor
```

Comment:
Ideally we shouldn't add any domain language from external projects so generalising this would be great. Some examples would be;

- `canEdit`
- `isStatic`